main/heart_stokes.cc

```diff
@@ -21,6 +23,15 @@
                                             "piDoMUS", \
                                             interface); \
   ParameterAcceptor::initialize(prm_file, pde_name+"_used.prm"); \
+  pidomus.lamdas.output_step = [&] (???)           \
+  {                                                                           \
+      auto &dof = interface->get_dof_handler();                               \
+      auto &tria = dof.get_triangulation();                                   \
+      double timestep, dt;                                                    \
+                                                                              \
+      ElasticProblem elastic_problem(timestep, dt);                           \
```

Comment:
`pidomus.lambdas.simulator->current_time` and `pidomus.lambdas.simulator->current_dt`

Why you need a signal here?